aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -909,48 +912,9 @@ module aptos_framework::stake {
             };
         };
 
-        // Sort the validator set, so we can ensure stable order and validator indices.
-        sort_validators(&mut next_epoch_validators);
-
-        // Store the validator index in the validator's ValidatorConfig resource, so we can look it up later.
-        // We can't rely on the validator set in CoreResource's ValidatorSet because some validators might choose
-        // to leave and get moved to pending_inactive later on.
-        let i = 0;
-        let len = vector::length(&next_epoch_validators);
-        while (i < len) {
-            let validator_info = vector::borrow_mut(&mut next_epoch_validators, i);
-            let pool_address = validator_info.addr;
-            let validator_config = borrow_global_mut<ValidatorConfig>(pool_address);
-            validator_config.validator_index = i;
-            validator_info.config.validator_index = i;
-            i = i + 1;
-        };
-
         validator_set.active_validators = next_epoch_validators;
     }
 
-    fun sort_validators(validators: &mut vector<ValidatorInfo>) {
```

Comment:
orthogonal to this but should we provide some sorting functionalities in Move as a part of our standard lib?